src/pages/settings/Profile/PersonalDetails/LegalNamePage.tsx

```diff
@@ -56,6 +58,8 @@ function LegalNamePage() {
             if (typeof values.legalLastName === 'string') {
                 if (!values.legalLastName) {
                     errors.legalLastName = translate('common.error.fieldRequired');
+                } else if (!isValidLegalName(values.legalLastName)) {
+                    addErrorMessage(errors, 'legalLastName', translate('privatePersonalDetails.error.hasInvalidCharacter'));
```

Comment:
We finally decided that the legal name can be anything but multiple spaces. We ended up removing these validations in 
https://github.com/Expensify/App/pull/76842